abi/abi.go

```diff
@@ -246,6 +252,13 @@ func (av *Value) Serialize() ([]byte, error) {
 		}
 
 		return []byte{b}, nil
+	case ProofsMode:
+		v, ok := av.Val.(proofs.Mode)
+		if !ok {
+			return nil, &typeError{proofs.TestMode, av.Val}
```

Comment:
BLOCKING: You've got an `Unknown` enum value in your `proofs.TestMode`, but you're defaulting to `TestMode` here. You should use `Unknown` instead.

Reply:
Resolved by `TestMode = 0`